stable/vpa/templates/recommender-podmonitor.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if and .Values.recommender.enabled .Values.recommender.podMonitor }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "vpa.fullname" . }}-recommender
```

Comment:
Do we need to add the ability to add labels to the pod monitor? If I remember correctly, the prometheus operator uses those to select which pod monitors to pull in.

Reply:
It's a valid point. I'll add them.

Reply:
Thanks!